MySQL not working on Velocity

**Describe the bug**
When using Triton with Velocity, Triton can not connect to MySQL server
And on paper it connected normally 

**To Reproduce**
1.Using Triton with Velocity
2.Config Triton to use MySQL
3.java.sql.SQLException: No suitable driver

**Expected behavior**
connect to MySQL server successfully

**Server information (please complete the following information):**
 - Server Minecraft Version: 1.18.2
 - Client Minecraft Version (if using ViaVersion or something similar): 1.18.2
 - BungeeCord (specify if it's a fork) Velocity 3.1.2
 - Triton Version: 3.8.0

**Additional context**
`29.06 03:25:41 [Server] INFO [triton]: [DEBUG] Reading config.yml file...
29.06 03:25:41 [Server] INFO [com.rexcantor64.shaded.hikari.HikariDataSource]: HikariPool-1 - Starting...
29.06 03:25:41 [Server] ERROR [triton]: Failed to connect to database, falling back to local storage!
29.06 03:25:41 [Server] INFO java.lang.RuntimeException: Failed to get driver instance for jdbcUrl=jdbc:mysql://********:3306/triton?useSSL=false&useUnicode=yes&characterEncoding=UTF-8
29.06 03:25:41 [Server] INFO at com.rexcantor64.shaded.hikari.util.DriverDataSource.<init>(DriverDataSource.java:114) ~[?:?]
29.06 03:25:41 [Server] INFO at com.rexcantor64.shaded.hikari.pool.PoolBase.initializeDataSource(PoolBase.java:331) ~[?:?]
29.06 03:25:41 [Server] INFO at com.rexcantor64.shaded.hikari.pool.PoolBase.<init>(PoolBase.java:114) ~[?:?]
29.06 03:25:41 [Server] INFO at com.rexcantor64.shaded.hikari.pool.HikariPool.<init>(HikariPool.java:108) ~[?:?]
29.06 03:25:41 [Server] INFO at com.rexcantor64.shaded.hikari.HikariDataSource.<init>(HikariDataSource.java:81) ~[?:?]
29.06 03:25:41 [Server] INFO at com.rexcantor64.triton.storage.MysqlStorage.load(MysqlStorage.java:67) ~[?:?]
29.06 03:25:41 [Server] INFO at com.rexcantor64.triton.Triton.setupStorage(Triton.java:147) ~[?:?]
29.06 03:25:41 [Server] INFO at com.rexcantor64.triton.Triton.reload(Triton.java:106) ~[?:?]
29.06 03:25:41 [Server] INFO at com.rexcantor64.triton.VelocityMLP.reload(VelocityMLP.java:60) ~[?:?]
29.06 03:25:41 [Server] INFO at com.rexcantor64.triton.Triton.onEnable(Triton.java:95) ~[?:?]
29.06 03:25:41 [Server] INFO at com.rexcantor64.triton.VelocityMLP.onEnable(VelocityMLP.java:39) ~[?:?]
29.06 03:25:41 [Server] INFO at com.rexcantor64.triton.plugin.VelocityPlugin.onEnable(VelocityPlugin.java:38) ~[?:?]
29.06 03:25:41 [Server] INFO at com.rexcantor64.triton.plugin.Lmbda$3.execute(Unknown Source) ~[?:?]
29.06 03:25:41 [Server] INFO at com.velocitypowered.proxy.event.UntargetedEventHandler$VoidHandler.lambda$buildHandler$0(UntargetedEventHandler.java:47) ~[velocity.jar:3.1.2-SNAPSHOT (git-e8bf6ab5-b161)]
29.06 03:25:41 [Server] INFO at com.velocitypowered.proxy.event.VelocityEventManager.fire(VelocityEventManager.java:598) ~[velocity.jar:3.1.2-SNAPSHOT (git-e8bf6ab5-b161)]
29.06 03:25:41 [Server] INFO at com.velocitypowered.proxy.event.VelocityEventManager.lambda$fire$5(VelocityEventManager.java:479) ~[velocity.jar:3.1.2-SNAPSHOT (git-e8bf6ab5-b161)]
29.06 03:25:41 [Server] INFO at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
29.06 03:25:41 [Server] INFO at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
29.06 03:25:41 [Server] INFO at java.lang.Thread.run(Thread.java:833) [?:?]
29.06 03:25:41 [Server] INFO Caused by: java.sql.SQLException: No suitable driver
29.06 03:25:41 [Server] INFO at java.sql.DriverManager.getDriver(DriverManager.java:299) ~[java.sql:?]
29.06 03:25:41 [Server] INFO at com.rexcantor64.shaded.hikari.util.DriverDataSource.<init>(DriverDataSource.java:106) ~[?:?]
29.06 03:25:41 [Server] INFO ... 18 more
29.06 03:25:41 [Server] INFO [triton]: Created a sample translations/default.json file since there were no translations
29.06 03:25:41 [Server] INFO [triton]: Loaded local storage manager
29.06 03:25:41 [Server] INFO [triton]: Successfully setup the language manager! 2 languages and 0 language items loaded!`

--- a/core/src/main/java/com/rexcantor64/triton/storage/MysqlStorage.java
+++ b/core/src/main/java/com/rexcantor64/triton/storage/MysqlStorage.java
@@ -57,6 +57,7 @@ public class MysqlStorage extends Storage {
     public void load() {
         this.ipCache = new IpCache();
 
+        config.setDriverClassName("com.mysql.cj.jdbc.Driver");
         config.setJdbcUrl("jdbc:mysql://" + this.host + ":" + this.port + "/" + this.database + "?useSSL=false" +
                 "&useUnicode=yes&characterEncoding=UTF-8");
         config.setUsername(user);

--- a/triton-velocity/build.gradle
+++ b/triton-velocity/build.gradle
@@ -13,6 +13,9 @@ dependencies {
     compileOnly 'com.velocitypowered:velocity-api:3.1.0'
     annotationProcessor 'com.velocitypowered:velocity-api:3.1.0'
 
+    // FIXME in the future change this to dynamic download and loading
+    implementation 'mysql:mysql-connector-java:8.0.30'
+
     implementation 'org.bstats:bstats-velocity:3.0.0'
 }
 

